docs-web/src/main/webapp/src/partial/docs/settings.workflow.edit.html

```diff
@@ -48,10 +48,12 @@ <h2 ng-show="!isEdit()" translate="settings.workflow.edit.add_workflow_title"></
                     <span class="input-group-addon">
                       <span class="fas fa-exchange-alt" ng-if="step.type == 'APPROVE'"></span>
                       <span class="fas fa-check-circle" ng-if="step.type == 'VALIDATE'"></span>
+                      <span class="far fa-file" ng-if="step.type == 'RESUME-REVIEW'"></span>
                     </span>
                     <select class="form-control" name="type-{{ $index }}" ng-model="step.type" ng-change="updateTransitions(step)" required>
                       <option value="APPROVE">{{ 'settings.workflow.edit.type_approve' | translate }}</option>
                       <option value="VALIDATE">{{ 'settings.workflow.edit.type_validate' | translate }}</option>
+                      <option value="RESUME-REVIEW">{{ 'settings.workflow.edit.type_resumereview' | translate }}</option>
```

Comment:
The step code should be `RESUME_REVIEW` (with `_` instead of `-`) to match the API format specified in #14.

```suggestion
                      <span class="far fa-file" ng-if="step.type == 'RESUME_REVIEW'"></span>
                    </span>
                    <select class="form-control" name="type-{{ $index }}" ng-model="step.type" ng-change="updateTransitions(step)" required>
                      <option value="APPROVE">{{ 'settings.workflow.edit.type_approve' | translate }}</option>
                      <option value="VALIDATE">{{ 'settings.workflow.edit.type_validate' | translate }}</option>
                      <option value="RESUME_REVIEW">{{ 'settings.workflow.edit.type_resumereview' | translate }}</option>
```